cf-custom-resources/package-lock.json

```diff
@@ -1,8 +1,10148 @@
 {
   "name": "ecs_cli_custom_resources",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Do we still want to change all these dependencies?

Reply:
I believe so -- I ran `npm install` to make sure the package-lock.json would be up to date first. Perhaps you can pull down the branch and see what happens for you, since you mentioned not getting changes to this file when running make recipes?

Reply:
OK apparently the `"dependencies"` field is used only for `lockfileVersion: 1`, and the dependency values in this new `packages` section seem to be identical to what we have under `dependencies` so we should be fine!